rts/Lua/LuaBitOps.cpp

```diff
@@ -57,11 +58,8 @@ static inline unsigned int luaL_checkuint(lua_State* L, int index)
 /*** Returns the bitwise OR of all arguments. Only use up to 24 bit integers.
  *
  * @function math.bit_or
- * @param number a1
- * @param number a2
- * @param number? a3
- * @param number? an
- * @return number i
+ * @param ... integer
```

Comment:
This obscures types somewhat.

Reply:
Actually the types are right, but it should require two minimum args. Will update.

![image](https://github.com/user-attachments/assets/c9919270-6634-4604-bf7b-c417e8b2cf6f)

Reply:
This particular function doesn't actually require any args. What I mean is that AFAICT "..." is the entirety of the type and "integer" is just the name of the variable(s). This may well be fine given the function is a bit operator but it's something to keep in mind if there are more of those untyped "..." vars elsewhere.

Reply:
The annotation is correct, there is no name parameter in the case of varargs.

Example: https://luals.github.io/wiki/annotations/#param

You can see in my screenshot that LLS is correctly validating the args.